src/nvim/fileio.c

```diff
@@ -1992,13 +1992,15 @@ readfile (
 }
 
 #ifdef OPEN_CHR_FILES
-/*
- * Returns TRUE if the file name argument is of the form "/dev/fd/\d\+",
- * which is the name of files used for process substitution output by
- * some shells on some operating systems, e.g., bash on SunOS.
- * Do not accept "/dev/fd/[012]", opening these may hang Vim.
- */
-static int is_dev_fd_file(char_u *fname)
+/// Returns true if the file name argument is of the form "/dev/fd/\d\+",
+/// which is the name of files used for process substitution output by
+/// some shells on some operating systems, e.g., bash on SunOS.
+/// Do not accept "/dev/fd/[012]", opening these may hang Vim.
+///
+/// @param fname The file name to check
+///
+/// @return true if the file name argument is of the form "/dev/fd/\d\+",
+static bool is_dev_fd_file(char_u *fname)
```

Comment:
@justinmk Actually, should such PRs also modify the function attributes if they modify a function signature?